config/manager/manager.yaml

```diff
@@ -29,9 +29,11 @@ spec:
         - --enable-leader-election
         image: controller:latest
         name: manager
-        env:
-          - name: WATCH_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.annotations['olm.targetNamespaces']
+        resources:
+          limits:
+            cpu: 100m
+            memory: 30Mi
+          requests:
+            cpu: 100m
+            memory: 20Mi
```

Comment:
I don't think we want this added. Might break the operator pod in certain environments. Let the users decide when to set this configuration.